### ISSUE TITLE
KDE Plasma 5

### ISSUE THREAD

**Issue by nuclearsandwich** [nuclearsandwich/surface3-archlinux#15]
I was testing KDE Plasma 5 on the Surface using [this Manjaro pre-release](https://manjaro.github.io/Manjaro-KDE-Plasma-0.9.0-pre1-edition-released/) and so far it's pretty... pretty. I'm here for the first time since the early KDE 3 days in search of a flexible touch / tiling hybrid environment but this is also the most stable I've gotten Linux on this hardware yet. I plan to try a 3.19 rc kernel for the typecover drivers but I'm typing this on a bluetooth keyboard with a trackpoint, so I'm pretty darn happy. I'll update with Plasma / Plasma Active notes as I go.

**Comment by ianks** [comment]
Which virtual keyboard do you use with Plasma 5?

**Comment by nuclearsandwich** [comment]
I haven't really. Didn't use it long enough to bother trying. Are there any decent ones besides onboard?

**Comment by DonHugo69** [comment]
I liked onboard very much, I just followed Vistaus tutorial found here:  [Click](https://github.com/Vistaus/surface3-arch-antergoslinux/issues/5)

Other alternatives might be [Florence] (http://florence.sourceforge.net/english.html) or more [here] (http://florence.sourceforge.net/english/alternatives.html)

But I prefer Gnome on the surface.

**Comment by nuclearsandwich** [comment]
Yeah, I completely eschewed the "it's also a tablet" aspect of the Surface even on Windows. The SP3 is just big enough and heavy enough and hot enough that it's pretty lousy compared to an iPad.

**Comment by ianks** [comment]
@nuclearsandwich I have not found that to be the case. I generally only use the tablet features for watching TV and reading, however.
